src/yadmark/benchmark/runner.py

```diff
@@ -120,6 +120,11 @@ def log(self, t, o, _pdf, me, ext):
             esfs = []
 
             obs_kins = kins.copy()
+
+            # skip if it is empty
+            if len(me[sf]) == 0:
```

Comment:
ehm, I guess during debugging I hit that corner case and it is technically not forbidden to ask for an observable with 0 ESFs ... which however will result here `if "y" in me[sf][0]:` in a KeyError; so it's just the coverage of a corner case

Reply:
Ok, I guess we should really validate our runcards.

However, for the empty case is better to produce a graceful trivial output, rather than failing validation. So this is just fine.

Thanks :)